camel.js

```diff
@@ -738,18 +750,22 @@ app.get('/rss', function (request, response) {
     }
     response.type('application/rss+xml');
 
-    if (typeof(renderedRss.date) === 'undefined' || new Date().getTime() - renderedRss.date.getTime() > 3600000) {
+    var previouslyRendered = renderedRss[request.hostname];
```

Comment:
At this level, it's almost certainly unlikely that `request` is undefined but you do have this nice safety check (https://github.com/cliss/camel/blob/master/camel.js#L186) that could be thrown in here too.

Reply:
Good point. However:
* `request` is given to me from express; if it ain't there, some shit has really broken
* I don't think it's that big a deal for the RSS feeds anyway; if they break, they break

Good spot either way. :)